compiler/modules/front.syntax/tests/testPreparse.bal

```diff
@@ -22,7 +22,6 @@ function testPreparse(string src, PreparseTestResult expected, int minLookahead,
     TokenizerState state = tok.save();
 
     test:assertEquals(tok.current(), "(");
-    check tok.advance();
```

Comment:
Why is this changed needed?

Reply:
Since `tok.advance()` was moved into `preparseParenTypeDesc` to accommodate for the `savePreparseRestore` helper function, we don't need to call it again here.